addons/medical_gui/gui.hpp

```diff
@@ -210,6 +210,7 @@ class ACE_Medical_Menu {
     class controlsBackground {
         class Title: RscText {
```

Comment:
We can just drop this bar altogether. Looks cleaner without it.

Reply:
The title bar is the drag handle for the window. May or may not be worth losing that. 
Side note, I was maybe going to look at preserving the window position through closing and reopening, to make dragging more useful.

Reply:
...the menu can be dragged?